libraries/protocol/include/scorum/protocol/scorum_operations.hpp

```diff
@@ -202,15 +202,10 @@ struct transfer_operation : public base_operation
     string memo;
 
     void validate() const;
+    
     void get_required_active_authorities(flat_set<account_name_type>& a) const
     {
-        if (amount.symbol != VESTS_SYMBOL)
```

Comment:
why you delete currency check?

Reply:
Because in transfer_operation::validate: 
FC_ASSERT(amount.symbol != VESTS_SYMBOL, "transferring of Scorum Power (STMP) is not allowed.");